src/components/molecules/Common/plugin/builtin/widgets/menu.tsx

```diff
@@ -77,16 +77,20 @@ const Menu: WidgetComponent<Property, PluginProperty> = ({ property }) => {
   const handleClick = useCallback(
     (b: Button | MenuItem) => () => {
       const t = "buttonType" in b ? b.buttonType : "menuType" in b ? b.menuType : undefined;
-      if (t === "link") {
-        const link = "buttonLink" in b ? b.buttonLink : "menuLink" in b ? b.menuLink : undefined;
-        window.open(link, "_blank", "noopener");
-      } else if (t === "menu") {
+      if (t === "menu") {
         setVisibleMenuButton(v => (v === b.id ? undefined : b.id));
         return;
       } else if (t === "camera") {
         const camera =
           "buttonCamera" in b ? b.buttonCamera : "menuCamera" in b ? b.menuCamera : undefined;
         setCamera(camera);
+      } else {
+        let link = "buttonLink" in b ? b.buttonLink : "menuLink" in b ? b.menuLink : undefined;
+        const splitLink = link?.split("/");
+        if (splitLink?.[0] !== "http:" && splitLink?.[0] !== "https:") {
+          link = "https://" + link;
+        }
+        window.open(link, "_blank", "noopener");
```

Comment:
```suggestion
        const link = "buttonLink" in b ? b.buttonLink : "menuLink" in b ? b.menuLink : undefined;
        if(!link)return;
        if (!/https?:\/\/[A-Za-z0=9_\-\.]*).test(link) {
          link = "https://" + link;
        }
        window.open(link, "_blank", "noopener");
```

Reply:
However, do we support the string without `http` or `https`? We never know if the linked website support TLS.

If we don't support

```
const link = "buttonLink" in b ? b.buttonLink : "menuLink" in b ? b.menuLink : undefined;
if(!link || !/https?:\/\/[A-Za-z0=9_\-\.]*).test(link))return;
window.open(link, "_blank", "noopener");
```

is also fine.

Reply:
I did this because without the full url window.open will append the url to the current location. like localhost/www.google.ca